Student.java

```diff
@@ -0,0 +1,115 @@
+package com.company;
+
+
+import java.util.Comparator;
+import java.util.List;
+
+
+public class Student {
+
+
+    private int id;
+    private String fName;
+    private String lName;
+    private int age;
+
+    public Student(int id, String firstName, String lastName, int ageFull) {
+        this.id = id;
+        this.fName = firstName;
+        this.lName = lastName;
+        this.age = ageFull;
+    }
+
+    public void printStudent() {
+        System.out.println("id\tFirst Name\tLast Name\tAge");
+        System.out.println(id + "\t" + fName + "\t" + lName + "\t" + age);
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getfName() {
+        return fName;
+    }
+
+    public String getlName() {
+        return lName;
+    }
+
+
+    public int getAge() {
+        return age;
+    }
+
+    public boolean studNameContains() {
+        String firstLetter = getfName();
+        boolean contains = false;
+        if (firstLetter.startsWith("T") || firstLetter.startsWith("t")) {
+            contains = true;
+        } else contains = false;
+        return contains;
+    }
+
+    @Override
+    public String toString() {
+        return "Student{" +
+                "id=" + id +
+                ", fName='" + fName + '\'' +

```

Comment:
fName, lName. I can only guess what is this. Better not to simplify smth when it's not needed.
First Name, Last Name - looks much better when printing